docker-for-ibm-cloud/dtr-ibm-cos.md

```diff
@@ -108,4 +228,22 @@ Before you begin:
 3. Follow the OS-specific instructions to [configure your host](/datacenter/dtr/2.4/guides/user/access-dtr/) to use the DTR CA.
 
 ## Verify that DTR is running
+
 Verify that DTR is set up properly by [pulling and pushing an image](/datacenter/dtr/2.4/guides/user/manage-images/pull-and-push-images/).
+
+## Delete IBM Cloud Object Storage
+
+1. Log in to your [IBM Cloud infrastructure account](https://control.softlayer.com/).
+
+2. Select **Storage** > **Object**.
+
+3. For the object storage **Account Name** that you want to delete, click the cancel icon.
+
+   > Tip to find your object storage account name
```

Comment:
Maybe "Find your object storage account name". This will be visually rendered as a note, so users will know that this is a tip.